lib/broadway_rabbitmq/producer.ex

```diff
@@ -703,6 +703,12 @@ defmodule BroadwayRabbitMQ.Producer do
   defp handle_connection_failure(state, reason) do
     _ = Logger.error("Cannot connect to RabbitMQ broker: #{inspect(reason)}")
 
+    :telemetry.execute(
+      [:broadway_rabbitmq, :amqp, :connection, :exception],
```

Comment:
```suggestion
      [:broadway_rabbitmq, :amqp, :connection_failure],
```

Reply:
My idea was to follow the same signature of events, which contains 4 elements. My initial plan was the one you propose.

Reply:
Those other events are spans, this one is not, so it is fine as is I think. /cc @whatyouhide